projects/canopy/src/styles/variables/_colors.scss

```diff
@@ -1,4 +1,6 @@
 :root {
+  --color-serenity: #faf7f5;
+
```

Comment:
why the line break?

Reply:
The other variables below seem to be grouped into shades of the same colour, separated by a line break, so I decided to put this in it's own group.

Thinking about that though, below we have a group of whites, greys and blacks, wonder if we could put it there?

```
--color-black: #000;
  --color-charcoal: #333;
  --color-battleship-grey: #575756;
  --color-taupe-grey: #929292;
  --color-platinum: #e3e3e3;
  --color-white-smoke: #f6f6f6;
  --color-white: #fff;
```

Reply:
@andy-polhill  I moved that var down to the section mentioned above